### PR TITLE
more api tests

### LINKED ISSUE #12: Bootstrap acceptance tests
Basic acceptance tests for ensuring the brute-force works as designed

#### Ensuring other users with the same IP can login
- ensure `user1` with `ip1` is blocked after x failed attempts
- ensure `user2` with `ip1` can still login

#### Ensuring that same user with different IP can login
- ensure `user1` with `ip1` is blocked after x failed attempts 
- ensure `user1` with `ip2` is allowed to login

Note - some things we might used during acceptance testing:
`X-Forwarded-For` Header would allow us to set an arbitrary IP in the scenario above

### PR DESCRIPTION
**Pull request by individual-it**
last part to fix #12

### CODE REVIEW

**Comment by individual-it** [comment]
drone cannot pass till next qa build

**Comment by phil-davis** [comment]
Needs step:
```
@Given user :arg1 has sent HTTP method :arg2 to URL :arg3 with password :arg4
```
to come in next core QA tarball

File: .drone.yml [hunk 1]
```diff
@@ -86,6 +86,9 @@ pipeline:
       - PLATFORM=Linux
       - BEHAT_SUITE=${BEHAT_SUITE}
     commands:
+      - cd /var/www/owncloud
+      - su-exec www-data php occ config:system:set trusted_proxies 1 --value=`ip addr show eth0 | awk '$1 == "inet" {gsub(/\\/.*$/, "", $2); print $2}'`
```
Comment (phil-davis) [inline .drone.yml]: @individual-it do you want to keep the ``inet`` output?
I guess it does not hurt so in future when IP-based tests start failing we can see what the drone interfaces look like?

Comment (individual-it) [reply .drone.yml]: for local drone I had to user `eth1` for drone-ci `eth0` so leaving both here

**Pull request merged**